jest.config.base.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   },
   testEnvironment: 'node',
   moduleDirectories: ['node_modules', './'],
-  modulePaths: ['node_modules', './']
+  modulePaths: ['node_modules', './'],
+  workerThreads: true
```

Comment:
In order to fix 
- #138 

- https://github.com/jestjs/jest/issues/11617#issuecomment-1458155552 was suggested

Reply:
The upstream issue was finally closed
- https://github.com/jestjs/jest/issues/11617

so you _might_ be able to remove `workerThreads` once the fix is published